lib/_stream_readable.js

```diff
@@ -48,12 +48,15 @@ function prependListener(emitter, event, fn) {
   // userland ones.  NEVER DO THIS. This is here only because this code needs
   // to continue to work with older versions of Node.js that do not include
   // the prependListener() method. The goal is to eventually remove this hack.
-  if (!emitter._events || !emitter._events[event])
+  if (!emitter._events || !emitter._events.has(event)) {
```

Comment:
Ping @mcollina @nodejs/streams ... please review this

Reply:
FWIW, I don't think this is correct but I need some input on what the current relationship between the npm package & node is. That original block of code seems to only be there defensively for the npm version, I don't actually know if it's relevant in the current version of Node.